standard-covoiturage_openapi.yaml

```diff
@@ -117,6 +117,105 @@ paths:
+      operationId: getDriverRegularTrips
+      parameters:
+        - $ref: '#/components/parameters/departureLat'
+        - $ref: '#/components/parameters/departureLng'
+        - $ref: '#/components/parameters/arrivalLat'
+        - $ref: '#/components/parameters/arrivalLng'
+        - $ref: '#/components/parameters/departureTimeOfDay'
+        - $ref: '#/components/parameters/timeDelta'
+        - $ref: '#/components/parameters/departureRadius'
+        - $ref: '#/components/parameters/arrivalRadius'
+        - $ref: '#/components/parameters/maxDepartureDate'
+        - $ref: '#/components/parameters/count'
+      responses:
+        200:
+          description: Ok. Request processed successfully.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  allOf:
+                    - $ref: '#/components/schemas/DriverTrip'
+                    - type: object
+                      properties:
+                        schedules:
+                          type: array
+                          items:
+                            $ref: '#/components/schemas/Schedule'
+
+  /passenger_regular_trips:
+    get:
+      tags:
+      - Search
+      summary: Search for matching passenger regular trips.
+      description: Route used to retrieve a collection of passenger regular trips matching the provided criteria.
+      operationId: getPassengerRegularTrips
+      parameters:
+        - $ref: '#/components/parameters/departureLat'
+        - $ref: '#/components/parameters/departureLng'
+        - $ref: '#/components/parameters/arrivalLat'
+        - $ref: '#/components/parameters/arrivalLng'
+        - $ref: '#/components/parameters/departureTimeOfDay'
+        - $ref: '#/components/parameters/timeDelta'
+        - $ref: '#/components/parameters/departureRadius'
+        - $ref: '#/components/parameters/arrivalRadius'
+        - $ref: '#/components/parameters/maxDepartureDate'
+        - $ref: '#/components/parameters/count'
+      responses:
+        200:
+          description: Ok. Request processed successfully.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  allOf:
+                    - $ref: '#/components/schemas/PassengerTrip'
+                    - type: object
+                      properties:
+                        schedules:
+                          type: array
+                          items:
+                            $ref: '#/components/schemas/Schedule'
+
+  /journeys:
+    get:
+      tags:
+      - Search
+      summary: Get journeys by their ids.
+      description: Route used to retrieve journeys using their identifiers.
+      operationId: getJourneys
+      parameters:
+        - in: query
+          name: ids[]
+          required: true
+          schema:
+            type: array
+            explode: true
```

Comment:
I'm not an expert of OpenAPI, but this gives me an error in my OpenAPI editor. Shouldn't "explode" be at the same level as schema instead of being a child ? (cf section 4.7.12.2 here : https://spec.openapis.org/oas/v3.0.1#fixed-fields-9)

Reply:
@adelcasse you are right, I must have missed it, it is fixed now.